packages/services/f-http/src/createClient.js

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import defaultOptions from './defaultOptions';
+
+let _configuration = null;
+let _axiosInstance = null;
+
+/**
+    * Get a resource
+    * @param {string} resource - The resource to post (URL)
+    * @return {object} - Returns data from response
+*/
+const get = async resource => {
+    try {
+        const response = await _axiosInstance.get(resource);
+
+        return response.data;
```

Comment:
Potential null exception if the response isn't a 2xx.

Reply:
Would be handled by the try-catch and throw to the caller who should handle it